File: specification/src/main/java/ru/dopaminka/specification/RunCucumberTest.java
package ru.dopaminka.specification;

import org.junit.runner.RunWith;

import io.cucumber.junit.Cucumber;
import io.cucumber.junit.CucumberOptions;

@RunWith(Cucumber.class)
@CucumberOptions(
        plugin = {"pretty"},
        features = {"src/main/res/features/*"},
        glue = {"ru.dopaminka.specification.steps.*"}
)
public class RunCucumberTest {
}

File: specification/src/main/java/ru/dopaminka/specification/RunCucumberTest.kt
package ru.dopaminka.specification

import io.cucumber.junit.Cucumber
import io.cucumber.junit.CucumberOptions
import org.junit.runner.RunWith

@RunWith(Cucumber::class)
@CucumberOptions(
    plugin = ["pretty"],
    features = ["src/main/raw/features"],
    glue = ["ru.dopaminka.specification.steps"]
)

class RunCucumberTest {
}
